plot.go

```diff
@@ -43,6 +42,7 @@ type brailleCell struct {
 type Plot struct {
 	*tview.Box
 	data           [][]float64
+	maxVal         float64
```

Comment:
The max data value was calculated in several places. And I would have had to calculated it again for this change. So, I figured it was better to do it once when the data was set.